hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ldb/ValueSchema.java

```diff
@@ -99,11 +96,11 @@ public Void call() throws Exception {
     return null;
   }
 
-  public static boolean getValueFields(String dbPath, Map<String, Object> valueSchema, int d, String table,
-                                       String dnDBSchemaVersion) {
+  public boolean getValueFields(String dbPath, Map<String, Object> valueSchema, int d, String table,
+                                       String dndbschemaversion) {
```

Comment:
Instead of using slightly different name to avoid `'dnDBSchemaVersion' hides a field`, let's remove parameters that were required only because the method was `static`.

```java
  public boolean getValueFields(String dbPath, Map<String, Object> valueSchema) {
```

(needs corresponding changes in caller and where parameters are used)